apps/explorer/src/pages/validator/DelegationAmount.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { SUI_TYPE_ARG } from '@mysten/sui.js';
+
+import { CoinFormat, useFormatCoin } from '~/hooks/useFormatCoin';
+import { Text } from '~/ui/Text';
+
+export function DelegationAmount({ amount }: { amount?: bigint | number }) {
```

Comment:
We have a lot of components that are just format coin + two text nodes (in this PR there's also StakeColumn that looks nearly identical), I wonder if these have consistent enough styling to unify these?